counterpartylib/lib/micropayments/__init__.py

```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
```

Comment:
hmm, probably remove these headers... or at least the copyright line and the license line, because if this is becoming part of the counterparty-lib source, the copyright line is listed in our `LICENSE` file, and the License line is not really necessary

looking into it further, ...obviously I am not a lawyer, but it appears there are two main ways to do this. first is you grant CP a irrevocable license to use the code and you keep the copyright, the 2nd is that the copyright for this code is assigned to CP... see http://producingoss.com/en/copyright-assignment.html

I'd prefer the second (as the code is becoming part of counterparty-lib, it keeps things tighter) but am open to discussion here